client/superdesk.config.js

```diff
@@ -62,7 +62,9 @@ module.exports = function(grunt) {
         
         search_cvs: [
             {id: 'belga-keywords', name:'Belga keywords', field: 'subject', list: 'belga-keywords'},
-            {id: 'services-products', name:'Packages', field: 'subject', list: 'services-products'}
+            {id: 'services-products', name:'Packages', field: 'subject', list: 'services-products'},
+            {id: 'sources', name:'Sources', field: 'subject', list: 'sources'},
+            {id: 'languages', name:'Language', field: 'subject', list: 'languages'}
```

Comment:
I think the field should be `source` and `language`

Reply:
We have CVs `sources` and `languages`, I think it should work.